scala-package/core/src/main/scala/org/apache/mxnet/Shape.scala

```diff
@@ -28,6 +30,7 @@ class Shape(dims: Traversable[Int]) extends Serializable {
   }
 
   def apply(dim: Int): Int = shape(dim)
+  def get(dim: Int): Int = apply(dim)
```

Comment:
why is a get method taking a parameter?

Reply:
Collection.get(i) for Collection[i] in JVM world

Reply:
got it. shape probably should have extended a collection.

Reply:
Don't see any advantage to make Shape as a collection, what I meant is, get(idx) is a code convention for get element of an index. Even though we make Shape extend Collection, but which? Collection in Java, or in Scala? which Collection? we should design. We can leave it later, otherwise this will be experimental.